doc/source/ecosystem.rst

```diff
@@ -149,13 +149,36 @@ for pandas ``display.`` settings.
 qgrid is "an interactive grid for sorting and filtering
 DataFrames in IPython Notebook" built with SlickGrid.
 
-`Spyder <https://github.com/spyder-ide/spyder/>`__
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`Spyder <https://www.spyder-ide.org/>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Spyder is a cross-platform PyQt-based IDE combining the editing, analysis,
+debugging and profiling functionality of a software development tool with the
+data exploration, interactive execution, deep inspection and rich visualization
+capabilities of a scientific environment like MATLAB or Rstudio.
+
+Its `Variable Explorer <https://docs.spyder-ide.org/variableexplorer.html>`__
+allows users to view, manipulate and edit pandas Index, DatetimeIndex, Series,
+and DataFrame objects with a full GUI, including the ability to
+sort by column, copy and modify individual values, display a "heatmap"
+per-column or globally, convert column data types and more, all with user- or
```

Comment:
Yup, as I mentioned above I though it might be a little long and was prepared to cut it some if requested. I cut it down from 22 lines to 15, a reduction of essentially 1/3, and matching the length of the Jupyter section. Is that good enough?